Libraries/DeprecatedPropTypes/DeprecatedTextInputPropTypes.js

```diff
@@ -41,56 +41,128 @@ module.exports = {
     'characters',
   ]): React$PropType$Primitive<'none' | 'sentences' | 'words' | 'characters'>),
   /**
-   * Determines which content to suggest on auto complete, e.g.`username`.
-   * To disable auto complete, use `off`.
+   * Specifies autocomplete hints for the system, so it can provide autofill. On Android, the system will always attempt to offer autofill by using heuristics to identify the type of content.
```

Comment:
Rephrasing @yungsters 's internal comment: 
You may want to make a PR for the deprecated [prop-types package](https://github.com/facebook/prop-types)